composer/loggers/wandb_logger.py

```diff
@@ -14,22 +16,25 @@
 from composer.core.types import Logger, State, StateDict
 from composer.utils import dist, run_directory
 
+__all__ = ["WandBLogger"]
+
 
 class WandBLogger(LoggerCallback):
     """Log to Weights and Biases (https://wandb.ai/)
 
     Args:
-        log_artifacts (bool, optional): Whether to log artifacts (default: ``False``)
+        log_artifacts (bool, optional): Whether to log artifacts (Default: ``False``)
         log_artifacts_every_n_batches (int, optional): Interval at which to upload
-            artifcats to wandb from the `run_directory`. On resnet50, a 22% regression
-            was realized when logging and uploading artifacts, so it is recommended to
-            do so infrequently. Only applicable when `log_artifacts` is True
-            (default: ``100``)
+            artifacts to wandb from the ``run_directory``. Logging very frequently (e.g.
+            on every batch) can substantially slow down training, so we recommend doing so
+            infrequently. Only applicable when `log_artifacts` is True (default: ``100``).
         rank_zero_only (bool, optional): Whether to log only on the rank-zero process (default: ``False``).
-            When logging artifacts, it is highly recommended to log on all ranks. Artifacts from ranks 1+
-            will not be stored, which may discard pertinent information. For example, when using Deepspeed
-            ZeRO, it would be impossible to restore from checkpoints without artifacts from all ranks.
-        init_params (Dict[str, Any], optional): Parameters to pass into :meth:`wandb.init`.
+            When logging artifacts, it is highly recommended to log on all ranks.
+            Artifacts from ranks 1+ will not be stored, which may discard pertinent
+            information. For example, when using Deepspeed ZeRO, it would be impossible to
+            restore from checkpoints without artifacts from all ranks.
+        init_params (Dict[str, Any], optional): Parameters to pass into :meth:`wandb.init`
```

Comment:
the :meth:`wandb.init` doesn't actually resolve in the documentation, since its external. 
```suggestion
        init_params (Dict[str, Any], optional): Parameters to pass into ``wandb.init``
```

Reply:
Is there a reason PyTorch and Python standard library references resolve but wandb don't? Can we get wandb references to resolve?